src/structures/Guild.js

```diff
@@ -326,7 +326,7 @@ class Guild extends Base {
     }
 
     /**
-     * The vanity URL code of the guild, if any
+     * The vanity URL code of the guild, if any. This only includes the invite code; not the entire URL
```

Comment:
Proposing different wording:

```suggestion
     * The vanity invite code of the guild, if any
```

You don't need to specify that it's not the entire URL if you don't mention URL in the first place.